README.md

```diff
@@ -18,6 +18,7 @@ Add the plugin, reporter and reporter configuration in your `karma.conf.js`.
   plugins: ['karma-remap-istanbul'],
   reporters: ['progress', 'karma-remap-istanbul'],
   remapIstanbulReporter: {
+    remapOptions: {} //additional remap options
```

Comment:
Aren't you missing a comma at the end of this line (even though it's just a README)?